### PR TITLE
feat:  suport kubevirt nic hotplug

### PR DESCRIPTION
**Pull request by watermelon-brother**
### What type of this PR
- Features

kubevirt hotplug interface will cause kube-ovn create veth-pair.

### CODE REVIEW

**Comment by oilbeater** [comment]
The lint error should be fixed

File: pkg/daemon/ovs_linux.go [hunk 1]
```diff
@@ -191,6 +191,10 @@ func generateNicName(containerID, ifname string) (string, string) {
 	if ifname == "eth0" {
 		return fmt.Sprintf("%s_h", containerID[0:12]), fmt.Sprintf("%s_c", containerID[0:12])
 	}
+	if strings.HasPrefix(ifname, "pod") {
```
Comment (oilbeater) [inline pkg/daemon/ovs_linux.go:194]: can you give an comment about why in some scenario the ifname will has prefix `pod` and exceed the length.

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
Thanks! @watermelon-brother

**Comment by bobz965** [comment]
backport "if" name starts with pod logic to release 1.11

**Pull request merged**